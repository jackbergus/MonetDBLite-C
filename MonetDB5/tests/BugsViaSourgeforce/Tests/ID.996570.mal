# Monet Mil to Mal compiler
# Copyright (c) 2001-2004, CWI. All rights reserved.

#Predefined code segment
#The compiler can not guarantee an accurate compilation,
#because MIL unlike MAL is a dynamically typed language.
#A few guidelines to maximize usefullness.
#- make the type of variables explicit, in particular 'nil'
#- upon advice of M2m to remap identifiers, change it in your program directly
#use the "mil_schema" for additional support routines
#	io.print("var a:=new(void,int);");
	a := new(nil:void,nil:int);
#	io.print("a.insert(nil,1);");
	aa_a := insert(a,nil,1);
#	io.print("a.insert(nil,2);");
	ba_a := insert(a,nil,2);
#	io.print("a.print();");
	ca_a := print(a);
#	io.print("var b:=new(oid,int);");
	b := new(nil:oid,nil:int);
#	io.print("b.insert(3@0,3);");
	da_a := insert(b,3@0,3);
#	io.print("b.insert(4@0,4);");
	ea_a := insert(b,4@0,4);
#	io.print("b.print();");
	fa_a := print(b);
#	io.print("var c:=a.copy().access(BAT_WRITE);;");
	BAT_WRITE:= mil.take("BAT_WRITE");
	ga_a := copy(a);
	c := access(ga_a,BAT_WRITE);
#	io.print("c.print();");
	ha_a := print(c);
#	io.print("var d:=c.insert(b);");
	d := insert(c,b);
#	io.print("d.print();");
	ia_a := print(d);
#	io.print("a.seqbase(0@0);");
#	io.print("a.print();");
	ka_a := print(a);
#	io.print("c:=a.copy().access(BAT_WRITE);");
	la_a := copy(a);
	c := access(la_a,BAT_WRITE);
#	io.print("c.print();");
	ma_a := print(c);
#	io.print("d:=c.insert(b);");
	d := insert(c,b);
#	io.print("d.print();");
	na_a := print(d);
#	io.print("b:=new(oid,int);");
	b := new(nil:oid,nil:int);
#	io.print("b.insert(2@0,3);");
	oa_a := insert(b,2@0,3);
#	io.print("b.insert(4@0,4);");
	pa_a := insert(b,4@0,4);
#	io.print("b.print();");
	qa_a := print(b);
#	io.print("c:=a.copy().access(BAT_WRITE);");
	ra_a := copy(a);
	c := access(ra_a,BAT_WRITE);
#	io.print("c.print();");
	sa_a := print(c);
#	io.print("d:=c.insert(b);");
	d := insert(c,b);
#	io.print("d.print();");
	ta_a := print(d);
