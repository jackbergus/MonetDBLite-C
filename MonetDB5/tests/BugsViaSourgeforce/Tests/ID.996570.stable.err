stderr of test 'ID.996570` in directory 'monetdb5/tests/BugsViaSourgeforce` itself:


# 17:00:22 >  
# 17:00:22 >  Mtimeout -timeout 60 Mserver "--config=/var/tmp/_PREFIX__GNU_32_d_--enable-debug_/etc/monet.conf" --debug=10 --set "monet_mod_path=/var/tmp/_PREFIX__GNU_32_d_--enable-debug_/lib/MonetDB" --set "gdk_dbfarm=/var/tmp/_PREFIX__GNU_32_d_--enable-debug_/var/MonetDB/dbfarm" --set "sql_logdir=/var/tmp/_PREFIX__GNU_32_d_--enable-debug_/var/MonetDB/log" --set mapi_port=46100 --set sql_port=57494 --set monet_prompt= --trace --dbname=mTests_tests_BugsViaSourgeforce  < ID.996570.milS
# 17:00:22 >  


# 17:00:23 >  
# 17:00:23 >  Done.
# 17:00:23 >  

