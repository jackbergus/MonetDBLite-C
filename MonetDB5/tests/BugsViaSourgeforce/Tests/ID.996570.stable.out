stdout of test 'ID.996570` in directory 'monetdb5/tests/BugsViaSourgeforce` itself:


# 17:00:22 >  
# 17:00:22 >  Mtimeout -timeout 60 Mserver "--config=/var/tmp/_PREFIX__GNU_32_d_--enable-debug_/etc/monet.conf" --debug=10 --set "monet_mod_path=/var/tmp/_PREFIX__GNU_32_d_--enable-debug_/lib/MonetDB" --set "gdk_dbfarm=/var/tmp/_PREFIX__GNU_32_d_--enable-debug_/var/MonetDB/dbfarm" --set "sql_logdir=/var/tmp/_PREFIX__GNU_32_d_--enable-debug_/var/MonetDB/log" --set mapi_port=46100 --set sql_port=57494 --set monet_prompt= --trace --dbname=mTests_tests_BugsViaSourgeforce  < ID.996570.milS
# 17:00:22 >  

# Monet Database Server V4.3.19
# Copyright (c) 1993-2004, CWI. All rights reserved.
# Compiled for i686-pc-linux-gnu/32bit; dynamically linked.
# Visit http://monetdb.cwi.nl/ for further information.
var a:=new(void,int);
a.insert(nil,1);
a.insert(nil,2);
a.print();
#-----------------#
# h	tmp_29	  # name
# void	int	  # type
#-----------------#
[ nil,	  1	  ]
[ nil,	  2	  ]
var b:=new(oid,int);
b.insert(3@0,3);
b.insert(4@0,4);
b.print();
#-----------------#
# h	tmp_30	  # name
# oid	int	  # type
#-----------------#
[ 3@0,	  3	  ]
[ 4@0,	  4	  ]
var c:=a.copy().access(BAT_WRITE);;
c.print();
#-----------------#
# h	tmp_28	  # name
# void	int	  # type
#-----------------#
[ nil,	  1	  ]
[ nil,	  2	  ]
var d:=c.insert(b);
d.print();
#-----------------#
# h	tmp_28	  # name
# void	int	  # type
#-----------------#
[ nil,	  1	  ]
[ nil,	  2	  ]
[ nil,	  3	  ]
[ nil,	  4	  ]
a.seqbase(0@0);
a.print();
#-----------------#
# h	tmp_29	  # name
# void	int	  # type
#-----------------#
[ 0@0,	  1	  ]
[ 1@0,	  2	  ]
c:=a.copy().access(BAT_WRITE);
c.print();
#-----------------#
# h	tmp_31	  # name
# void	int	  # type
#-----------------#
[ 0@0,	  1	  ]
[ 1@0,	  2	  ]
d:=c.insert(b);
d.print();
#-----------------#
# h	tmp_31	  # name
# oid	int	  # type
#-----------------#
[ 0@0,	  1	  ]
[ 1@0,	  2	  ]
[ 3@0,	  3	  ]
[ 4@0,	  4	  ]
b:=new(oid,int);
b.insert(2@0,3);
b.insert(4@0,4);
b.print();
#-----------------#
# h	tmp_28	  # name
# oid	int	  # type
#-----------------#
[ 2@0,	  3	  ]
[ 4@0,	  4	  ]
c:=a.copy().access(BAT_WRITE);
c.print();
#-----------------#
# h	tmp_30	  # name
# void	int	  # type
#-----------------#
[ 0@0,	  1	  ]
[ 1@0,	  2	  ]
d:=c.insert(b);
d.print();
#-----------------#
# h	tmp_30	  # name
# oid	int	  # type
#-----------------#
[ 0@0,	  1	  ]
[ 1@0,	  2	  ]
[ 2@0,	  3	  ]
[ 4@0,	  4	  ]
